feature-group-definitions/css-modules.yml

```diff
@@ -0,0 +1,9 @@
+name: CSS module scripts
```

Comment:
I know we've used the term "JavaScript module" for the JS counterpart, but I chose to use the "scripts" suffix here because there was a lot of discussion around this name in the feature proposal originally.